### PR TITLE
Add natural earth coastlines

### PR DESCRIPTION
**Pull request by zerebubuth**
Refs #203.

Add Natural Earth coastline shapefiles to the repository, update scripts to import them and the queries to fetch them. This requires some code changes, see mapzen/tilequeue#28 and mapzen/TileStache#49.

@rmarianski could you review, please?

### CODE REVIEW

**Comment by nvkelso** [comment]
We should also reference the Natural Earth lake polygons already loaded and convert them to boundaries in the query, too?

**Comment by zerebubuth** [comment]
Thanks, @nvkelo, I completely forgot about lakes and playas. Added those in 06bcfe8. See the screenshot below for how this looks with a very simple style at zoom 8 (all NE).

water_boundaries_z8

In the upper left you can see the shores of lake Erie and some of the minor lakes which surround it. Here's how NYC looks zoomed in a bit more to 10 (mix of NE & OSM data):

water_boundaries_z10

I'm not sure whether the detail visible in the landuse areas and all the small lakes means we should look at reducing detail at this zoom, but that's a matter for another issue.

**Comment by zerebubuth** [comment]
Note: the cut across the Hudson to the north of that last screenshot doesn't seem to be the same issue as the previous one. I'm investigating further.

**Comment by zerebubuth** [comment]
Okay, the bug above is orthogonal to this PR, and should be fixed in mapzen/TileStache#50.

@nvkelso how's it looking?

**Comment by nvkelso** [comment]
The new 1:10,000,000 coastline file (merc) has an error linking the Siberian coastline from the far right side of the map to the far left side. The offending vertex on the far left side should be deleted.

screen shot 2015-09-15 at 07 55 08

File: queries/water.jinja2 [hunk 1]
```diff
@@ -54,6 +99,26 @@ FROM ne_10m_playas
...
+    NULL AS name,
+    NULL AS area,
+    'playa' AS kind,
+    'naturalearthdata.com' AS source,
+    st_boundary(the_geom) AS __geometry__,
+    gid AS __id__,
+    'yes' AS boundary
+
+FROM
+{% if 4 <= zoom < 7 %}
```
Comment (nvkelso) [inline queries/water.jinja2]: We should be consistent with the other water levels of detail: 
{% if 2 <= zoom < 5 %}

File: queries/water.jinja2 [hunk 2]
```diff
@@ -54,6 +99,26 @@ FROM ne_10m_playas
...
+    ne_50m_playas
+{% elif 7 <= zoom < 9 %}
```
Comment (nvkelso) [inline queries/water.jinja2]: We should be consistent with the other water levels of detail: 
{% elif 5 <= zoom < 9 %}

File: queries/water.jinja2 (hunk 1 above)
Comment (zerebubuth) [inline queries/water.jinja2]: Yup, well spotted, thanks! Looks like the _playas_ only kick in at zoom 4, though. Otherwise, fixed in 65f727d.

**Comment by nvkelso** [comment]
> Here's how NYC looks zoomed in a bit more to 10 (mix of NE & OSM data):

@zerebubuth: Can you explain this more, why are we seeing a mix of Natural Earth and OSM data at zoom 10 – shouldn't that be OSM only?

**Comment by nvkelso** [comment]
Once the 1:10m coastline file has been fixed, I think this is ready to go.

**Comment by zerebubuth** [comment]
> Can you explain this more, why are we seeing a mix of Natural Earth and OSM data at zoom 10 – shouldn't that be OSM only?

@nvkelso, you're right - I just saw `source != "openstreetmap.org"` and thought it was NE, but it's actually OSM we're using for the ocean tiles. Forget I said that about the mix of NE & OSM.

**Comment by rmarianski** [comment]
This looks good to me too :zap:

**Comment by zerebubuth** [comment]
> The new 1:10,000,000 coastline file (merc) has an error linking the Siberian coastline from the far right side of the map to the far left side. The offending vertex on the far left side should be deleted.

Hmm... that came straight from `ogr2ogr`. Must have been a precision / wraparound error. Fixed in 00429901 by removing all the new segments it had created by intersecting with the existing coastline.

**Comment by nvkelso** [comment]
@zerebubuth I'm still seeing the rogue coastline in the SHP in the new ZIP. Can you include a screenshot here showing your fix?

**Comment by nvkelso** [comment]
Adding issue to reducing LOD at some zooms (less water, less landuse):

* https://github.com/mapzen/vector-datasource/issues/213

**Comment by zerebubuth** [comment]
@nvkelso Here's the screenshot. Not sure why that's useful, though - if you're seeing the issue still, with the latest version of the file, then we need to figure out why you're seeing it and I'm not.

fixed_dateline_line

**Comment by nvkelso** [comment]
Weird. Everything checks out today, nothing more to see here :)

**Pull request merged**